src/EnergyPlus/CMakeLists.txt

```diff
@@ -928,23 +928,22 @@ if(LINK_WITH_PYTHON)
   add_custom_command(
     TARGET energyplusapi
     POST_BUILD # TODO: I don't think we want to quote the generator expression
-               # here
     COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/cmake/PythonCopyStandardLib.py" "$<TARGET_FILE:energyplusapi>" "python_standard_lib")
 endif()
 
 if(BUILD_PACKAGE)
   # if we are building package, we need to drop in some API/Plugin stuff
   if(LINK_WITH_PYTHON)
-    # we'll want to grab the standard lib for python plugins TODO: I don't think
-    # we want to quote the generator expression
+    # we'll want to grab the standard lib for python plugins
+    # TODO: I don't think we want to quote the generator expression
```

Comment:
Unrelated but I think we can remove the TODO. https://cmake.org/pipermail/cmake/2014-June/057768.html
```suggestion
```